analysis/PE_AMI cox model code stata.do

```diff
@@ -0,0 +1,91 @@
+* Script to run PE and ATE outcomes in the electively unvaccinated population in STATA
+* Electively unvaccinated is a whole population sample
+
+clear
+cd "C:\Users\rd16568\OneDrive - University of Bristol\Documents\CONVALESCENCE\CVD stata analysis"
+import delim using "input_sampled_data_ate_covid_pheno_hospitalised_vaccinated_reduced_time_periods.csv"
```

Comment:
Replace with `import delim using "output/input_sampled_data_ate_covid_pheno_hospitalised_vaccinated_reduced_time_periods.csv"` because the working directory will be 'post-covid-vaccinated' in the real data.